HospitalWebApp/Controllers/ExaminationController.cs

```diff
@@ -63,6 +64,10 @@ public IActionResult ScheduleExamination(ScheduledExaminationDTO dto)
         [HttpPost]
         [Route("recommend")]
         public IActionResult RecommendExamination(RecommendationRequestDto dto)
-            => Ok(_examinationService.Recommend(dto));
+        {
+            Debug.Write(Ok(_examinationService.Recommend(dto)[0].RoomId)+ "sobica");
+            return Ok(_examinationService.Recommend(dto));
+        }
+       // => Ok(_examinationService.Recommend(dto));
```

Comment:
I guess that you didn't changed logic in this method, just added debug message?

Reply:
Oh i forgot to delete that :D